river/View.zig

```diff
@@ -557,6 +556,28 @@ pub fn attachAfter(view: *Self, pending_state: *Output.PendingState, n: usize) v
     it.current.prev.?.insert(&view.pending_wm_stack_link);
 }
 
+/// Attach above or below the currently focused view
+pub fn attachRelative(view: *Self, pending_state: *Output.PendingState, mode: AttachRelativeMode) void {
+    var focus_stack_it = pending_state.focus_stack.iterator(.forward);
+    const focused_view = focus_stack_it.next();
+
+    if (focused_view == null) {
+        pending_state.wm_stack.append(view);
+        return;
+    }
+
+    var it = pending_state.wm_stack.iterator(.forward);
+    while (it.next()) |other| {
+        if (other == focused_view) {
+            switch (mode) {
+                .above => it.current.prev.?.insert(&view.pending_wm_stack_link),
+                .below => it.current.insert(&view.pending_wm_stack_link),
+            }
+            return;
+        }
+    }
```

Comment:
This appears to have the same issue of not respecting tags as the original patch.

Reply:
@ifreund 
It seems not to be necessary here from my testing.
Because we are inserting directly in the `wm_stack` without the detour over the index counting...

Reply:
Indeed, another benefit of having this logic all in one function!